packages/app/src/views/newBait.tsx

```diff
@@ -53,7 +53,7 @@ const StyledButton = styled.button`
 const NewBaitForm = () => {
 	const currentUser = useCurrentUser();
 	const userID = currentUser?.id;
-	const [baitID, setBaitID] = useState("");
+	const [baitID, setBaitID] = useState("651c1094231992254e2d4c3e");
```

Comment:
this was the fastest way to fix the bug, dont judge :DD

Reply:
This broke #42 now if you add new post it causes the posts page to not work with error message of 

> Cannot return null for non-nullable field Post.bait.
I'll approve nevertheless because this problem was caused by my placeholder coding and make necessary to  #42 before merging it.